modules/light-clients/08-wasm/types/types_test.go

```diff
@@ -160,3 +157,15 @@ func TestWasmTestSuite(t *testing.T) {
 func getAltSigners(altVal *tmtypes.Validator, altPrivVal tmtypes.PrivValidator) map[string]tmtypes.PrivValidator {
 	return map[string]tmtypes.PrivValidator{altVal.Address.String(): altPrivVal}
 }
+
+// NewPathWithWasmTendermint constructs an endpoint for each chain using
+// the default values for the endpoints, expect for the client config, for
+// which it uses the Wasm Tendermint config, so that both endpoints use the
+// Tendermint Wasm light client contract.
+// Each endpoint is updated to have a pointer to the counterparty endpoint.
+func NewPathWithWasmTendermint(chainA, chainB *ibctesting.TestChain, codeHash []byte) *ibctesting.Path {
```

Comment:
I don't feel too strongly, but we use `NewTransferPath` in the transfer test suite. We could use `NewWasmTendermintPath` here to align

Reply:
If it's defined in a test file in 08-wasm. I think it's fine to simply call it `NewPath` as well

Reply:
It is specifically a wasm tendermint path though, it might be useful to have that clarification in the name (as opposed to some other wasm client implementation )

Maybe

```golang
path := wasm.NewTendermintPath(...)
```